src/main/java/minicraft/level/tile/OreTile.java

```diff
@@ -97,6 +98,9 @@ public void hurt(Level level, int x, int y, int dmg) {
 			} else {
 				level.setData(x, y, damage);
 			}
+			if (type.drop.equals(Items.get("gem"))){
+				AchievementsDisplay.setAchievement("minicraft.achievement.find_gem", true);
+			}
```

Comment:
Only changes like this would have been appropriate with the current title of the PR.